queries.sql

```diff
@@ -55,3 +55,45 @@ SELECT neutered, SUM(escape_attempts) AS total_attempts FROM animals GROUP BY ne
 SELECT species, MIN(weight_kg) AS minimum, MAX(weight_kg) AS maximum FROM animals GROUP BY species;
 -- What is the average number of escape attempts per animal type of those born between 1990 and 2000?
 SELECT species, AVG(escape_attempts) FROM animals WHERE date_of_birth BETWEEN '1990-01-01' AND '2000-12-31' GROUP BY species;
+
+
+-----------------------------------------------------
+--What animals belong to Melody Pond?
+SELECT a.id, a.name, o.full_name
+FROM animals a
+JOIN owners o ON a.owner_id = o.id WHERE o.full_name = 'Melody Pond';
+
+-----------------------------------------------------
+--List of all animals that are pokemon (their type is Pokemon)
+
+SELECT a.id, a.name, s.name
+FROM animals a
+JOIN species s on a.species_id = s.id WHERE s.name = 'Pokemon';
+
+-------------------------------------------------------
+--List all owners and their animals, remember to include those that don't own any animal.
+SELECT o.full_name, o.id, a.name FROM owners o
+LEFT JOIN animals a ON o.id = a.owner_id;
+
+-------------------------------------------------------
+-- How many animals are there per species?
+SELECT s.id, s.name as species, COUNT(a.id) as number_of_animals
+FROM animals a JOIN species s ON a.species_id = s.id
+GROUP BY s.id;
+-----------------------------------------------------------
+-- List all Digimon owned by Jennifer Orwell.
+SELECT a.id, a.name, o.full_name, s.name as species FROM animals a
+JOIN owners o ON a.owner_id = o.id
+JOIN species s ON a.species_id = s.id
+WHERE o.full_name = 'Jennifer Orwell' AND s.name = 'Digimon';
```

Comment:
- [ ] Great work implementing the change, but as the previous reviewer mentioned, the output result which is evident through the screenshot is wrong. Kindly update the screenshot as proof that this query returns the correct result. 👍 😎